opm/parser/eclipse/EclipseState/Summary/Summary.cpp

```diff
@@ -79,30 +82,114 @@ namespace Opm {
         return res;
     }
 
+    static inline std::array< int, 3 > dimensions( const EclipseGrid& grid ) {
+        return {
+            int( grid.getNX() ),
+            int( grid.getNY() ),
+            int( grid.getNZ() )
+        };
+    }
+
+    static inline std::array< int, 3 > getijk( const DeckRecord& record,
```

Comment:
It looks like this overload always gets `int` argument - so I guess there is no reason to make it optional? Instead of `init` I would prefer `offset`.